unity3d/Assets/Swrve/SwrveSDK/SwrveSDKImp.cs

```diff
@@ -1217,26 +1253,25 @@ protected virtual void ProcessCampaigns (Dictionary<string, object> root)
 
                     for (int i = 0, j = jsonCampaigns.Count; i < j; i++) {
                         Dictionary<string, object> campaignData = (Dictionary<string, object>)jsonCampaigns [i];
-                        SwrveCampaign campaign = SwrveCampaign.LoadFromJSON (this, campaignData, initialisedTime, swrveTemporaryPath);
-                        if (campaign.Messages.Count > 0) {
```

Comment:
Have you tested that it still works with a campaign with no messages? We used to have those back in the day (accidentally)